omc3/model/madx_macros/lhc.macros.run3.madx

```diff
@@ -1,23 +1,43 @@
 /*
 * Macros for runIII of the LHC using the new standard of the _op for the knobs used in operation.
+* If called after lhc.macros.madx these will override the previous definitions.
 */
 
 !requires general.macros.madx
 
 
+/*
+* Performs the matching of the LHC tunes.
+* This is adapted for Run 3 to use the *_op knobs
+* @param nqx: The horizontal tune to match to.
+* @param nqy: The vertical tune to match to.
+* @param beam_number: The beam to use, either 1 or 2.
+*/
+match_tunes(nqx, nqy, beam_number): macro = {
```

Comment:
this macro is actually correct, it should be called (and **not** the `match_tunes_ats` one)

Reply:
I thought we used the ATS macros in 2021 too?